### PR TITLE
remove Go generator plugin

### PR DESCRIPTION
**Pull request by noahdietz**
Remove gnostic-go-generator plugin from main repo in favor of https://github.com/googleapis/gnostic-go-generator

cc @glickbot

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,7 +8,7 @@ build:
 	cd apps/petstore-builder; go get; go install
 	cd plugins/gnostic-summary; go get; go install
 	cd plugins/gnostic-analyze; go get; go install
-	cd plugins/gnostic-go-generator; go get; go install
+	go get github.com/googleapis/gnostic-go-generator
```
Comment (timburks) [inline Makefile]: Is this needed now? Maybe not since we decided not to test the generator in the main repo.

File: README.md [hunk 2]
```diff
@@ -97,7 +97,7 @@ Protocol Buffer description. This is mainly for use in testing and debugging.
 that reports some basic information about an API. The "-" causes the plugin to 
 write its output to stdout.
 
-        go install github.com/googleapis/gnostic/plugins/gnostic-go-generator
+        go install github.com/googleapis/gnostic-go-generator
```
Comment (timburks) [inline README.md]: maybe this and the following line should go too

File: Makefile (hunk 1 above)
Comment (noahdietz) [reply Makefile]: I'll remove it

File: README.md (hunk 2 above)
Comment (noahdietz) [reply README.md]: Might be good to have an example, that's what this section of the readme is about. Should I switch it to one that exists in the repo? e.g. swift?

Happy to remove it altogether though.

Comment (noahdietz) [reply README.md]: Removed it an rephrased the section.

Comment (timburks) [reply README.md]: LGTM! IMO the Swift repo also needs to go into a separate repo.

**Comment by timburks** [comment]
Thanks, Noah!

**Pull request merged**